easybuild/scripts/rpath_args.py

```diff
@@ -52,6 +53,11 @@
 else:
     rpath_filter = None
 
+if not rpath_include:
+    rpath_include = []
+else:
+    rpath_include = rpath_include.split(',')
```

Comment:
positive logic is easier to follow (and maybe worth a comment to clarify possible `None` value):

```python
if rpath_include:
    rpath_include = rpath_include.split(',')
else:  # take into account that rpath_include may be None
    rpath_include = []